src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -261,13 +258,26 @@ applyCoinbase v logger dbEnv (Miner mid mks) reward@(ParsedDecimal d) pd parentH
     forkTime = vuln797FixDate v
     fork1_3InEffect = blockTime >= forkTime
     throwCritical = fork1_3InEffect || enfCBFailure
+
     blockTime = blockTimeOf pd
+        -- FIXME it should hold that @blockTime == _blockCreationTime parentHeader@, but
+        -- in some unit test runs that is not the case
```

Comment:
The Fixme here is correct. Let's check the unit tests

Reply:
Agree, let's standardize on parentBlock and stop using `PublicData` altogether, and fix the tests